src/tests/simulation/src/cluster.rs

```diff
@@ -434,17 +399,26 @@ impl Cluster {
         }
         std::env::set_var("RW_META_ADDR", meta_addrs.join(","));
 
-        let mut sql_endpoint = String::new();
-        let mut backend_args = if let Some(sqlite_data_dir) = conf.sqlite_data_dir.as_ref() {
-            sql_endpoint = format!(
-                "sqlite://{}stest-{}.sqlite?mode=rwc",
+        // FIXME: some tests like integration tests will run concurrently,
+        // resulting in connecting to the same sqlite file if they're using the same seed.
+        let file_path = if let Some(sqlite_data_dir) = conf.sqlite_data_dir.as_ref() {
```

Comment:
Will just (2) be sufficient?

Reply:
Reason being for (1), it's a convenient feature. Let's say if I realize some deterministic test is failing in main-cron, I can easily add some logs, and rerun the step, because the set of madsim seeds will always be fixed.

But now that it's changed to random, subsequent runs may not yield the same results anymore.

Reply:
Since integration tests will run in parallel, using the same seed may cause some tests to unexpectedly connect to the same SQLite file.